runtime/testy/studentCode.py

```diff
@@ -9,10 +9,28 @@ def autonomous_main():
   pass
 
 def teleop_setup():
-  pass
+  uids = Robot._getAllUIDs()
```

Comment:
I think so.
The "test" is "can we drive a robot". 
A separate branch is almost guaranteed to start diverging